x/datadeal/README.md

```diff
@@ -0,0 +1,52 @@
+## Data Deal Commands
+
+### Create a Deal
+```shell
+CREATOR=$(panacead keys show {your deal creator} -a)
+
+panacead tx datadeal create-deal --deal-file {your deal json file path} --from $CREATOR --chain-id {your chain ID}
+```
+Deal JSON File Example
+```json
+{
+  "data_schema": [
+    "https://www.json.ld"
+  ],
+  "budget": "10000000umed",
+  "max_num_data": 10000,
+  "trusted_data_validator": [
```

Comment:
```suggestion
  "trusted_data_validators": [
```

Reply:
Thanks! Fixed in [81e5590](https://github.com/medibloc/panacea-core/pull/333/commits/81e55900ff9d2df340f6736d2a6aa193277cad0b).